### PR TITLE
Notes/GitHub and markdown

### PR DESCRIPTION
**Pull request by AnjaSiewert**
Pls check

### CODE REVIEW

**Review by DominicWebDev — APPROVED** [review]
LGTM

File: css/css.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+p {
+    font-family: sans-serif;
```
Comment (Satttoshi) [inline css/css.md:2]: I dont like this font

Comment (DominicWebDev) [inline css/css.md:2]: font-familiy: ARIAL

File: html/html.md [hunk 2]
```diff
@@ -0,0 +1 @@
+<p>Hello World</p>
```
Comment (Satttoshi) [inline html/html.md:1]: Hello World please with "!"

Comment (DominicWebDev) [inline html/html.md:1]: World Hello

File: READ.ME [hunk 3]
```diff
@@ -0,0 +1 @@
+This is the README file for my first Git branch switch
```
Comment (Satttoshi) [inline READ.ME:1]: this is cool

**Review by DominicWebDev — APPROVED** [review]

**Review by ChristophObst — APPROVED** [review]

**Review by Satttoshi — COMMENTED** [review]
Beautiful

**Pull request merged**